.github/workflows/dbt_slim_ci.yml

```
@@ -0,0 +1,48 @@
+# .github/workflows/app.yaml
+name: dbt slim ci (in beta)
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - 'main'
+
+concurrency:
+  group: ${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  dbt-test:
+    runs-on: [ self-hosted, linux, spellbook ]
+    timeout-minutes: 60
+
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+
```

- name: Set environment variables
```suggestion
      - name: Set environment variables
        run: |
          echo "GIT_SHA=${GITHUB_SHA::7}" >> $GITHUB_ENV
```
I'm not sure if this works 😅